### PR TITLE
Merge branch 'chakra-uwp' of https://github.com/munyirik/node into chakra-uwp

### PR DESCRIPTION
**Pull request by munyirik**
- Update tests to check for 'chakra' instead of 'chakracore'
- Fix fs_unlink. The only code change is to add the SetFileAttributesW line. The rest of the changes in fs.c are formatting corrections.

### CODE REVIEW

**Comment by munyirik** [comment]
@kunalspathak could you take a look? thanks.

File: deps/uv/src/win/fs.c [hunk 1]
```diff
@@ -737,99 +737,101 @@ void fs__rmdir(uv_fs_t* req) {
...
-        OPEN_EXISTING,
-        FILE_FLAG_OPEN_REPARSE_POINT | FILE_FLAG_BACKUP_SEMANTICS,
-        NULL);
-
-    if (handle == INVALID_HANDLE_VALUE) {
-        SET_REQ_WIN32_ERROR(req, GetLastError());
-        return;
-    }
+  /* Remove read-only attribute before trying to delete */
+  SetFileAttributesW(pathw, GetFileAttributesW(pathw) & ~FILE_ATTRIBUTE_READONLY);
```
Comment (kunalspathak) [inline deps/uv/src/win/fs.c]: >SetFileAttributesW [](start = 2, length = 18)

Can you check the return value and if it failed, throw GetLastError() and return ?

File: test/parallel/test-util-log.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ var tests = [
   {input: 42, output: '42'},
   {input: function() {}, output: common.engineSpecificMessage({
     v8: '[Function: input]',
-    chakracore: '[Function: input]'
+    chakra: '[Function: input]'
```
Comment (kunalspathak) [inline test/parallel/test-util-log.js]: >chakra [](start = 4, length = 6)

I wanted to make it something like isChakraEngine() which would match both 'chakracore' and 'chakra', but I don't recall why we decided to go with exact match.

**Comment by kunalspathak** [comment]
:shipit:

File: deps/uv/src/win/fs.c (hunk 1 above)
Comment (munyirik) [inline deps/uv/src/win/fs.c]: yup. I can change that.

**Pull request merged**